src/lib/cache/subCache.cpp

```diff
@@ -211,6 +211,20 @@ void subCacheInit(bool multitenant)
 
 /* ****************************************************************************
 *
+* subCacheDisable -
+*
+*/
+#ifdef UNIT_TEST
+void subCacheDisable(void)
```

Comment:
I have need to add this function to avoid subCacheActive to be `true` during no-cache unit test. Maybe not very clean, but the UNIT_TEST define will ensure that this doesn't "leak" into contextBroker binary.

NTC (just informative)